app/views/foreman/smart_proxies/_content_tab.html.erb

```diff
@@ -0,0 +1,42 @@
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class=""><%= s_("Content View Name") %></th>
+      <th class=""><%= s_("Composite") %></th>
+      <th class=""><%= s_("Last Published") %></th>
+      <th class=""><%= s_("Hosts") %></th>
+      <th class=""><%= s_("Products") %></th>
+      <th class=""><%= s_("Yum repos") %></th>
+      <th class=""><%= s_("Docker repos") %></th>
+      <th class=""><%= s_("Packages") %></th>
+      <th class=""><%= s_("Errata") %></th>
+      <th class=""><%= s_("Puppet modules") %></th>
```

Comment:
`s_` does namespaced translations. Can be just `_` here as well. I can change it to `translate` if that's preferred.